lib/core/src/lib/viewer/components/viewer-render/viewer-render.component.spec.ts

```diff
@@ -70,7 +72,7 @@ describe('ViewerComponent', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [CoreTestingModule, MatButtonModule, MatIconModule],
+            imports: [NoopAnimationsModule, TranslateModule.forRoot(), CoreTestingModule, MatButtonModule, MatIconModule],
```

Comment:
CoreTestingModule already contains NoopAnimations and Translate, so you can omit those here

Reply:
done :)